langchain/src/vectorstores/vectara.ts

```diff
@@ -166,14 +175,18 @@ export class VectaraStore extends VectorStore {
   async similaritySearchWithScore(
     query: string,
     k = 10,
-    filter: VectaraFilter | undefined = undefined
+    filter: VectaraFilter | undefined = undefined,
```

Comment:
Looking at this now I realize this is not the ideal name for this variable since it's not a filter anymore, it's just a data structure with some variables that control the search, right? part of it is a filter but now there's also lambda and contextConfig. So perhaps update the name?

Reply:
So I actually tried adding another parameter to the function, but since the VectaraStore class extends VectorStore, I couldn't give the function different parameters than those of the base class. Instead, I just added the context config to the filter. Do you have any ideas on how to get around this